### DISCUSSION TITLE
Youtube Data api

### DISCUSSION THREAD

**Discussion by MrGrease** [googleapis/nodejs-pubsub#1567] [Q&A] [answered]
How would I use the node implementation of this to listen to pubsub notifications from the youtube api? Is this even possible?

**Comment by feywind** [comment, marked as answer]
Good question, and yes you can :) You can use it to listen to any Pub/Sub subscription. (You'd just need the name of the subscription, and follow the listenForMessages sample.) I'll add this to our list of end to end sample ideas.

**Reply by MrGrease** [reply]
Thanks for the response! What exactly would the name be in the case of the YoutubeData Api's pubsubhubbub notifications?
